overview.md

```diff
@@ -0,0 +1,58 @@
+Anbox Cloud provides a rich software stack to enable you to run Android in the cloud for all kinds of different uses cases. This page intends to give you an overview over the available variants and internal components which make Anbox Cloud.
+
+## Variants
+
+Anbox Cloud exists in two different variants which serve different purposes:
+
+**Anbox Cloud Appliance**
+
+The Anbox Cloud Appliance is a self-contained, single-machine deployment variant of Anbox Cloud. It uses a sensible standard configuration to hide the complexity and flexibility of Juju. In this way, the Anbox Cloud Appliance allows simple and fast development and prototyping.
+
+**Anbox Cloud**
+
+The regular Anbox Cloud uses [Juju](https://juju.is/) for deployment and operations. It provides rich features and is ready made for a large scale deployment.
+
+Which of both variants you choose depends on your needs. The appliance is well suited for quick prototyping and development or small scale deployments, whereas the regular Anbox Cloud is best to scale big.
+
+## Components
+
+Anbox Cloud is composed of many pieces interacting with one another. This page gives a summary of what those pieces are and how they interact with each other.
+
+### Core Stack
+
+Anbox Cloud takes care of all management aspects and provides both a fully functional implementation and an integration model to support existing infrastructure and service implementations.
+
+The following figure gives an overview over the different components and their responsibility within the core stack of Anbox Cloud.
+
+![anbox-core-overview|690x398](upload://x0budFKybsbYLhiplNhF4R91Ght.png)
+
+At the heart of Anbox Cloud sits the **Anbox Management Service (AMS)**. **AMS** has the job to handle all aspects of the application and container lifecycle (including application and image updates) while ensuring high density, performance and fast container startup times.
+A developer or system administrator will manage **AMS** through the **command line interface (AMC)** or through custom-built tools interacting with the [**AMS REST API**](https://discourse.ubuntu.com/t/ams-rest-api-reference/17801).
+
+For example, a simple Android application testing service would provide a user-facing interface dealing with things like authentication and user management, and would communicate with the REST API to add applications or start and stop containers when a user asks to.
```

Comment:
This was "...communicate with the REST API to add applications, start and containers when a user asks to.". Is the update correct?

Reply:
Sounds good to me